examples/terraform-remote-exec-example/files/get-public-ip.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# For example purposes, print the public IP address of this instance
+echo $(wget -q -O- http://169.254.169.254/latest/meta-data/public-ipv4)
```

Comment:
Is the `echo` and subshell necessary? Won't calling `wget` directly produce the exact same output?